<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="DataBase">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Database</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/database.png</normaloff>images/database.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Connection">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Connection</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/connection.png</normaloff>images/connection.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Map">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Map</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/map.png</normaloff>images/map.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GetPacket">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GetPacket</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Massage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Packets</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_database_editor">
   <property name="text">
    <string>open database editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
